memberships/templates/base.html

```diff
@@ -23,7 +25,8 @@
 
 
         <nav class="navbar navbar-expand-lg">
-          <h1><a href="/" title="Home">Geek.Zone</a></h1>
+          <!-- <h1 class="logo"><a href="/" title="Home">Geek.Zone</a><span></span></h1> -->
```

Comment:
Could we remove this commented out line?

Reply:
sure